checkstyle.xml

```diff
@@ -101,7 +101,7 @@
     <!--module name="AvoidInlineConditionals"/-->
     <module name="CovariantEquals"/>
     <module name="EmptyStatement"/>
-    <module name="EqualsAvoidNull"/>
+    <!--<module name="EqualsAvoidNull"/>-->
```

Comment:
Checkstyle insists that this can cause null pointer exceptions, and I should flip the `.equals()`. I disagree.

```
if (lunch.equals("taco")) { ... }
```